src/bsoncxx/document/value.hpp

```diff
@@ -88,6 +125,31 @@ class BSONCXX_API value {
     ///
     element operator[](stdx::string_view key) const;
 
+    ///
+    /// Access the raw bytes of the underlying document.
+    ///
+    /// @return A (non-owning) pointer to the view's buffer.
```

Comment:
```suggestion
    /// @return A (non-owning) pointer to the value's buffer.
```

Reply:
Also, this pointer _is_ owning. So we should remove the _non-owning_ portion of the comment.

Reply:
Done!